src/TimeService.Models.Dto/Requests/HelpersModels/EditWorkTimeModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.JsonPatch;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.TimeService.Models.Dto.Requests.HelpersModels
+{
+    public class EditWorkTimeModel
```

Comment:
it is very strange model

Reply:
I passed parameters to the validator. Any idea how best?

Reply:
You can check jsonpatchdocument db model after map

Reply:
Profit? @djino97

Reply:
not need write not useless method

Reply:
I didn't understand how it would work ...